### PR TITLE
Add Lemmy

### PR DESCRIPTION
**Pull request by slothtown**

### CODE REVIEW

**Comment by Carlgo11** [comment]
I think this would be one of those [self hosted services](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md#self-hosted-services) that are excluded.

**Comment by slothtown** [comment]
I kind of disagree. Lemmy is part of the fediverse and works in a similar way to Mastodon, which isn't considered an excluded site. Both can be self hosted but it's definitely not mandatory, as one can also join a public instance. The reason I mentioned the version that an instance needs to be on is that 2fa is still a new feature in Lemmy and there might be some instances that haven't updated yet.

File: img/j/join-lemmy.org.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6 41.78-41.848s-18.705-41.848-41.78-41.848c-23.075 0-41.78 18.736-41.78 41.848zm-280.56 0c0 23.324 18.877 42.233 42.165 42.233 23.287 0 42.166-18.909 42.166-42.234 0-23.325-18.879-42.233-42.166-42.233-23.288 0-42.166 18.908-42.166 42.233z" style="display:inline;opacity:1;fill:#000;fill-opacity:1;stroke:#000;stroke-width:0;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" transform="translate(0 -26.067)"/><path d="m339.73 769.247-54.545 72.225m399.086-72.225 54.544 72.225M263.683 697.82l-87.755 35.824m579.857-35.824 87.756 35.824" style="display:inline;opacity:1;fill:none;stroke:#000;stroke-width:32;stroke-linecap:round;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" transform="translate(0 -26.067)"/><path d="M512 713.09c-45.863 0-75.13 31.845-74.14 71.1 1.07 42.513 32.468 71.104 74.14 71.104 41.674 0 74.052-32.996 74.142-71.103.094-39.269-28.277-71.101-74.141-71.101z" style="display:inline;opacity:1;fill:#fff;fill-opacity:1;stroke:#000;stroke-width:28;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" transform="translate(0 -26.067)"/></g></svg>
```
Comment (Carlgo11) [inline img/j/join-lemmy.org.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#fff" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="28" viewBox="0 0 1024 1024"><path d="M716.9 336.9a190.3 190.3 0 0 1 56.5-53.3c66.6-38.5 137-33.2 170 22.4 33.1 55.7 15 117.5-47.6 159.9a248 248 0 0 1-97.2 35.5M307 337a190.3 190.3 0 0 0-56.5-53.3c-66.6-38.5-137-33.2-170 22.4-33.1 55.7-15 117.5 47.6 159.9 32 19.2 63 30.9 97.2 35.5Z"/><path d="M801.2 550.8C812.7 401 720.6 295.9 512 299.3 303.4 302.7 213.7 402 222.8 550.8c8.6 142 176.8 246.4 288 246.4 111.4 0 279.5-104.5 290.4-246.4z"/><path fill="#000" d="M610.5 618.2a41.8 41.8 0 1 0 83.6 0 41.8 41.8 0 0 0-83.6 0zm-280.6 0a42.2 42.2 0 1 0 84.4 0 42.2 42.2 0 0 0-84.4 0z"/><path stroke-width="32" d="m339.7 743.2-54.5 72.2m399-72.2 54.6 72.2M263.7 671.8l-87.8 35.8m579.9-35.8 87.7 35.8"/><path d="M512 687c-45.9 0-75.1 31.9-74.1 71.1 1 42.5 32.4 71.1 74.1 71.1a73 73 0 0 0 74.1-71c.1-39.3-28.2-71.2-74.1-71.2z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thank you for your comment.
I see that you haven't added any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by slothtown** [comment]
I couldn't find a documentation so here are some screenshots. 
Screenshot from 2023-08-05 20-53-43
Screenshot from 2023-08-05 20-55-49
Screenshot from 2023-08-05 20-55-36

**Comment by slothtown** [comment]
One thing that is perhaps important to mention is that atm Lemmy doesn't ask you to confirm succesfully enabling 2fa with a one-time code, so it's possible to enable 2fa, not save the secret and then log out locking yourself out.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**